config/dark.in

```diff
@@ -0,0 +1,20 @@
+# DARK options
+
+config DARK
+	bool
+	prompt "Enable Dark Ages"
+	default n
+	help
+	  Select this to enable the Dark Ages mod
+
+config DARK_VERSION
+    prompt "Dark Ages version"
+	string
+    default "4_47_04_v39"
+
+config DARK_DROPBOX_URL
+    prompt "Dark Ages Download URL"
+	string
+    default "https://www.dropbox.com/s/zvc6f34y6jiy6j9/DarkAges_4_47_04_v39.zip"
```

Comment:
This zip file includes a whole copy of the windows version of Dwarf Fortress, given that this is a linux pack, should that be trimmed?

Reply:
That is the official download link from the mod author. They only release windows versions. So the only way to create a Linux pack for Dark Ages is to download the windows file and copy the raws and graphics into a Linux df Install.

That's what my PR does.

Reply:
Yeah I was thinking that it may be interesting to create another download link that contains a trimmed version of the original release, however now I think that your solution is better as maintaining a different release is more work and much more likely to fail.